lib/topological_inventory/ansible_tower/receptor/api_object.rb

```diff
@@ -29,7 +29,7 @@ def get
       end
 
       def post(data)
-        response = send_request(:post, endpoint, :data => data)
+        response = send_request(:post, endpoint, data)
```

Comment:
@lindgrenj6 @slemrmartin was line 32 `:data => data` in the cloud tower request? I'm interested why it was here if Tower didn't like the request. Unless the difference here is how the href_slug is handled vs the direct url.

Reply:
@syncrou 
https://github.com/ansible/ansible_tower_client_ruby/blob/47155754e93da0d00afaeaac1a54681270a52663/lib/ansible_tower_client/base_models/job_template.rb#L16

Reply:
ah, the main difference is, that it was intended that `AnsibleTower::Receptor::Template` will launch, but `AnsibleTowerClient::JobTemplate` is used instead (as a result of previous `find` method). 

Anyway the `params` are intended as string after `?` in href_slug (GET requests) and `:data` should be sent in the payload as "params" (it is intended for POST data, @mkanoor, right?)

Reply:
@slemrmartin We dont need anything with the href_slug. You can put all the parameters into params.